data/net/minecraft/advancements/critereon/DamageSourcePredicate.mapping

```diff
@@ -1,14 +1,24 @@
 CLASS net/minecraft/advancements/critereon/DamageSourcePredicate
```

Comment:
Please change `name` to `property` in:
- `getOptionalBoolean`
- `addOptionally`